src/components/Main.js

```diff
@@ -41,6 +42,13 @@ export default function Main({ onHandleSubmit }) {
     navigate("/flightplan");
   }
 
+  function selectDepartureRunway(runway) {
+    setDepartureRunway(`${runway}`);
```

Comment:
e.target.value is already a string. you don't have to template string here anymore :)